x11-drivers/nvidia-drivers/files/bin/install-nvidia

```diff
@@ -2,17 +2,23 @@
 
 . /usr/share/coreos/release
 NVIDIA_DRIVER_BASENAME="$1"
-
-emerge-gitclone
-emerge -gKv coreos-sources > /dev/null
-cp "/usr/lib64/modules/$(ls /usr/lib64/modules)/build/.config" /usr/src/linux/
-make -C /usr/src/linux modules_prepare
+KERNEL_NAME="$(ls /lib/modules)"
 
 cd "/nvidia/${NVIDIA_DRIVER_BASENAME}"
 ./nvidia-installer -s -n \
   --no-check-for-alternate-installs \
   --no-kernel-module-source \
+  --kernel-name="${KERNEL_NAME}" \
   --no-opengl-files \
   --no-distro-scripts \
-  --kernel-install-path="${PWD}" \
+  --no-systemd \
+  --skip-depmod \
+  --kernel-install-path="${PWD}/install-mod" \
   --log-file-name="${PWD}/nvidia-installer.log" || true
+
+mkdir -p /lib/modules/${KERNEL_NAME}/video
+mkdir -p "${PWD}"/install-mod
+cp "${PWD}"/kernel/*.ko "${PWD}"/install-mod
```

Comment:
`nvidia-installer` fails when it can't modprobe immediately, which means it always fails in the dev container. Depending on installer version, the modules are copied to `kernel-install-path` anyway, or not. So we have to assume the module files are in `$PWD/kernel/*.ko` and fetch them from there.